client/src/profile/Addresses.js

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from 'react';
+import AddressField from './AddressField';
+
+const Addresses = ({
+  addresses,
+  saveNewAddress,
+  updateAddressState,
+  addressFieldState,
+  removeAddress,
+ }) => (
+   <div>
+     <h3>Receive Alerts For Which Addresses?</h3>
+     {addresses.map(address => (
+       <div key={address.address}>
```

Comment:
Do you have a rule of thumb for how big a component should be? this file is a result of my pulling that component out of the ProfileContainer.